src/background.js

```diff
@@ -5,6 +5,12 @@ chrome.runtime.onInstalled.addListener(function () {
 // Set up an alarm to check Devin's status periodically
 chrome.alarms.create("checkDevinStatus", { periodInMinutes: 1 });
 
+// Initialize the storage for the last notification time and URL
+chrome.storage.local.set({
+  lastNotificationTime: null,
+  lastNotificationUrl: null,
+});
```

Comment:
User can open devin session in multple tab. I want to handle each tab seperately.